bundle-size-chart/cloud_build.yaml

```diff
@@ -0,0 +1,43 @@
+steps:
+  # The first two steps are required to identify and inject un-redacted secrets.
+- name: gcr.io/cloud-builders/npm
```

Comment:
nit: Indentation + 2 (here and every other step).

As a future task, Prettier [supports](https://github.com/ampproject/amphtml/blob/master/.vscode/settings.json) yaml auto-formatting.

Reply:
Done.